### PR TITLE
Modernises the Meta-class and changes its faction to NT

### PR DESCRIPTION
**Pull request by Zevotech**
## About The Pull Request
The Meta was a comfy little whiteship in dire need of fixing. It had tinyfans, light fixture frames and an uncomfortably cramped kitchen and botany area. This PR remaps it while keeping its soul intact, making it an old Nanotrasen vessel instead of an Indie.

NEW META (NO AREAS):
StrongDMM-2023-12-06 00 54 04

NEW META (AREAS):
StrongDMM-2023-12-06 00 54 00

NEW META (INGAME):
image

OLD META (NO AREAS):
StrongDMM-2023-12-05 17 42 10

## Why It's Good For The Game
I don't want this pretty good ship to be removed for being old and having tinyfans. Having another NT ship that fills a logistics/cargo niche is also pretty great.

## Changelog

:cl:
add: Remapped the Meta-class and changed its faction to NT
/:cl:

### CODE REVIEW

**Comment by Luftkommando** [comment]
firefox_iPH2WeBRkR

The APC isn't wired into the network. Cable shows unpowered in-game. Probably best to run it through the airlock instead of the wall.

**Comment by Zevotech** [comment]
lol holy shit how did I miss that. I'll fix it ASAP. oops! wallcable!

**Comment by Martinpachu** [comment]
The captain has an independant captain's outfit and not Nanotrasen.

**Comment by Zevotech** [comment]
Got it. Jobs were from before the outfit paths got updated so they slipped through.

**Comment by Martinpachu** [comment]
you can add more captain slots??

**Comment by Zevotech** [comment]
> you can add more captain slots??

oh my god this fucking ship

File: _maps/configs/nanotrasen_meta.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
 	"$schema": "https://raw.githubusercontent.com/shiptest-ss13/Shiptest/master/_maps/ship_config_schema.json",
 	"map_name": "Meta-class Freighter",
-	"prefix": "ISV",
+	"prefix": "NTSV",
 	"map_short_name": "Meta-class",
 	"description": "The Meta-class is a small freight vessel, and even before the ICW was a common sight on the Frontier as a tramp freighter, running independent contracts between the myriad outposts of the area (with, occasionally, some smuggling or mining on the side). Since the collapse of Nanotrasen’s logistics network in the Frontier region, Meta-classes operating in this capacity have exploded in popularity, and are likely to remain a very common sight wherever larger corporations such as Donk! Co. have yet to establish market dominance.",
```
Comment (Latentish) [inline _maps/configs/nanotrasen_meta.json]: This description is a little weird if we're making it a Nanotrasen ship.

**Review by Latentish — DISMISSED** [review]
Please kill the ORM

**Comment by Latentish** [comment]
Cool thanks merging now.

**Pull request merged**